components/director/internal/open_resource_discovery/validation.go

```diff
@@ -472,6 +475,9 @@ func validateEventInput(event *model.EventDefinitionInput, packagePolicyLevels m
 			return validateExtensibleField(value, event.OrdPackageID, packagePolicyLevels)
 		})),
 		validation.Field(&event.DocumentationLabels, validation.By(validateDocumentationLabels)),
+		validation.Field(&event.CorrelationIDs, validation.By(func(value interface{}) error {
+			return validateJSONArrayOfStringsMatchPattern(value, regexp.MustCompile(StringArrayElementRegex))
+		})),
```

Comment:
Line 760 and 761 should be changed because:
"For API Resource definition sap-sql-api-definition-v1: The apiProtocol MUST be set to sap-sql-api-v1."

Reply:
https://github.tools.sap/CentralEngineering/open-resource-discovery-specification/pull/578/files

Reply:
https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document#api-resource-definition